tiledb/sm/rest/rest_client.cc

```diff
@@ -690,6 +691,46 @@ RestClient::post_enumerations_from_rest(
       serialization_type_, returned_data);
 }
 
+std::string RestClient::post_query_plan_from_rest(
```

Comment:
Just a heads up, I noticed that there were test failures and its because you forgot the `ifdef`'ed out version of this function that just throws an exception about serialization not being enabled.

Reply:
Thanks for the failure investigation! It's now fixed